addon/components/power-select.hbs

```diff
@@ -36,6 +36,8 @@
       {{did-update this._performSearch this.searchText}}
       {{on "keydown" this.handleTriggerKeydown}}
       {{on "focus" this.handleFocus}}
+      {{on "mousedown" this.handleMousedown}}
```

Comment:
to allow for focusing the combobox (input) as soon as the user tabs to it, to reveal the combobox sooner (the audit asked to reveal it by default, but that seemed impossible)